schema-dex-amm-extended.graphql

```diff
@@ -652,6 +805,121 @@ type LiquidityPoolHourlySnapshot @entity {
 
   " Per-block reward token emission as of the current block normalized to a day (not hour), in USD value. This should be ideally calculated as the theoretical rate instead of the realized amount. "
   rewardTokenEmissionsUSD: [BigDecimal!]
+
+  ##### Account/Position Data #####
+
+  " Number of positions in this market "
+  positionCount: Int!
+
+  " Number of open positions in this market "
+  openPositionCount: Int!
+
+  " Number of closed positions in this market "
+  closedPositionCount: Int!
+}
+
+####################################
+##### Pool Tick-Level Metadata #####
+####################################
+
+type Tick @entity {
+  " { pool address }-{ tick index } "
+  id: Bytes!
```

Comment:
It’s probably still early to know, but do you think these ticks will be the way other concentrated liquidity DEXes will work?

Reply:
I think yes, but I am not certain.